src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -360,6 +576,9 @@ void ViewSceneParse(HandlerContext & ctx, const CommandData & req, GroupDataProv
 CHIP_ERROR StoreSceneParse(const FabricIndex & fabricIdx, const EndpointId & endpointID, const GroupId & groupID,
                            const SceneId & sceneID, GroupDataProvider * groupProvider)
 {
+    // Make SceneValid false for all fabrics before store scenes
+    ScenesServer::Instance().MakeSceneInvalidForAllFabrics(endpointID);
```

Comment:
It would,  since storing a Scene makes the new current scene that newly created scene (See appcluster 1.4.8.4) so the previous current scene and group should be invalidated.

Reply:
But current scene and group are per-fabric.  Storing a scene does not affect "current scene" or group _on other fabrics_.

Reply:
Changed to only invalidate on current fabric.